spec/unit/file_system_spec.rb

```diff
@@ -872,7 +872,6 @@ def increment_counter_in_multiple_processes(file, num_procs, options)
 
             # regardless of slash direction, return value is drive letter
             expanded = Puppet::FileSystem.expand_path(slash)
-            expect(expanded).to eq(ENV['SystemDrive'] + File::SEPARATOR)
             expect(expanded).to eq(File.expand_path(slash))
```

Comment:
the test feels a little self-referential, that `Puppet::FileSystem.expand_path(slash) == File.expand_path(slash)`. Maybe assert that there is a drive letter instead of assuming systemdrive?

```
expect(expanded).to match(/^[a-z]:/i)
```